lib/httpInterface.js

```diff
@@ -722,6 +722,10 @@ async function customErrorHandler(error, request, reply) {
     return
   }
 
+  if (error.message === 'body must NOT have additional properties') {
+    throw new Error(`${error.message}. Property "${error.validation[0].params.additionalProperty}" is not defined in validation schema`)
```

Comment:
Here we are showing only the first error, so that users can correct one error at time?

Reply:
yes, unfortunately Ajv for some reason gives only one property with that error at a time even if you have multiple additional properties